disassembler.asm

```diff
@@ -443,6 +444,14 @@ disasm_branch:
                 ; The address after the 0BRANCH is handled the same as a literal.
                 bra disasm_print_literal
 
+; DO handler
+disasm_do:
+                lda #'D'
+                jsr emit_a
+                lda #'O'
+                jsr emit_a
+                rts
+
```

Comment:
it didn't seem worth creating a whole new string for this but not sure if this is what you intended

Reply:
I think that's less bytes than adding a string and putting it into the lookup table, so that's fine.  It is what I was intending.  
It's technically not a JSR to `do` itself, but I don't think we need extra effort to attempt to make that clear.  You did put it in uppercase here, so that makes it a little bit distinct - even through Tali is not case sensitive, if someone did JSR to `do` then Tali would list that in lowercase as the words are lowercased in the dictionary.